app/_mesh_redirects

```diff
@@ -0,0 +1,21 @@
+# TODO: Automate this file's generation
+# Generated redirects
+/mesh/:version/overview/ /mesh/:version/overview/what-is-a-service-mesh 301
+/mesh/:version/installation/ /mesh/:version/installation/kubernetes 301
```

Comment:
Q about this one: why not redirect to the install landing page, ie `/mesh/:version/install/`?

https://deploy-preview-5079--kongdocs.netlify.app/mesh/latest/install/

Reply:
cc @lahabana

Reply:
Yes could work too